plasmapy/physics/transport/collisions.py

```diff
@@ -253,7 +256,10 @@ def _boilerPlate(T, particles, V):
     charges = [np.abs(p.charge) for p in particles]
 
     # obtaining reduced mass of 2 particle collision system
-    reduced_mass = atomic.reduced_mass(*particles)
+    if not np.isinf(mass_override):
+        reduced_mass = mass_override
```

Comment:
I would rename these to `mass = mass_overide` and `mass = atomic.reduced_mass(*particles)`